app/views/idv/phone/new.html.erb

```diff
@@ -65,6 +65,25 @@
         required: true,
         class: 'margin-bottom-4',
       ) %>
+  <%= render AlertComponent.new(
+        type: :warning,
+        class: 'margin-bottom-4',
+        text_tag: 'div',
```

Comment:
`text_tag` is only useful to override if the default paragraph tag wouldn't work, usually if you're rendering multiple paragraphs of text. Since we're not here, I think we want the default paragraph.

```suggestion
```